geocity/apps/submissions/tables.py

```diff
@@ -140,6 +140,12 @@ class GenericSubmissionTable(ColumnShiftTable):
         verbose_name=_("Entité administrative"),
         orderable=False,
     )
+    selected_price = tables.TemplateColumn(
+        verbose_name=_("Tarif"),
+        template_name="tables/_submission_selected_price.html",
+        attrs=ATTRIBUTES,
+        orderable=False,
+    )
```

Comment:
What happens is no payment is configured ? Should not this field only be visible when a payment is configured.

Reply:
It's displayed as `—`, like for the submissions without dates:
![image](https://user-images.githubusercontent.com/4549666/235603360-350207ad-4b85-4c11-bbda-bb915304a379.png)

Reply:
Okay for you @monodo?